ui/app/routes/vault/cluster/clients/counts.ts

```diff
@@ -116,7 +116,11 @@ export default class ClientsCountsRoute extends Route {
       activityError,
       config,
       // activity.startTime corresponds to first month with data, but we want first month returned or requested
-      startTimestamp: this.paramOrResponseTimestamp(params?.start_time, activity?.byMonth[0]?.timestamp),
+      // unless no months present, then we can fallback to response's start time
+      startTimestamp: this.paramOrResponseTimestamp(
```

Comment:
Fixes issue where dates are not set on load for enterprise when no data is available